sqlmesh/core/model/meta.py

```diff
@@ -78,6 +78,7 @@ class ModelMeta(_Node):
     enabled: bool = True
     physical_version: t.Optional[str] = None
     gateway: t.Optional[str] = None
+    optimize: t.Optional[bool] = None
```

Comment:
Should this be part of the model's data hash?

Reply:
Does the metadata hash include it automatically or should I include it manually?

Reply:
you need to include it manually